gaffer-as-a-service/gaas-rest/src/main/resources/application.properties

```diff
@@ -13,9 +13,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-jwt.secret=javainuse
+
 gaffer.namespace=kai-dev
 ingress.suffix=apps.k8s.example.com
-admin.username=javainuse
-admin.password=$2a$10$slYQmyNdGzTn7ZLBXBChFOC9f6kFjAqPhccnP6DxlWXx2lPk1C3G6
 management.endpoints.web.exposure.include=prometheus,metrics
+spring.profiles.active=cognito
```

Comment:
IS the default cognito then?

Reply:
Yes although happy for it to be JWT if preferred?

Reply:
> Nice changes! The linting and tidying is appreciated, however, in the future it would be nicer to split the formatting and cognito stuff into separate PRs please.
> Just a few questions then ready to approve

@t92549 
Thank you. Yeah sorry about that appreciate its not ideal 😢 . If there needs to be a large scale reformat like this again I'll make sure its in its own PR.

Reply:
No this is fine just wondering, will it work properly for the mock middleware stuff?

Reply:
Yeah when you run the mock middleware stuff in the UI it doesn't actually connect to any backend so should be fine.